keyboard/src/modules/Keyboard.rb

```diff
@@ -1504,6 +1483,43 @@ def write_udev_rule
     end
   end
 
+  # Checks if the graphical environment is being executed remotely using
+  # "ssh -X"
+  def x11_over_ssh?
+    display = Builtins.getenv("DISPLAY") || ""
+    display.split(":")[1].to_i >= 10
```

Comment:
@lslezak even if you are right and there is no hostname, changing the way we do the check sounds risky with no clear benefit. I'll change the call to ```getenv``` but keep the old approach.